src/components/AddTypeWithInput.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+    <v-row align="center" justify="center">
+        <v-col cols="7" class="pa-0 pb-2">
+            <!-- <v-row align="center" justify="center"> -->
+            <v-text-field v-model="kind" label="Rodzaj" filled hide-details></v-text-field>
+            <!-- </v-row> -->
+        </v-col>
+        <v-col cols="4" class="pa-2">
+            <Counter :quantity="quantity" :plus="plus" :minus="minus" />
+        </v-col>
+    </v-row>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from "vue-property-decorator";
+
+import Counter from "@/components/Counter.vue";
+
+@Component({
+    components: {
+        Counter
+    }
+})
+export default class AddTypeWithInput extends Vue {
+    @Prop() readonly updateSupplies!: any;
+    @Prop() readonly type!: string;
+    @Prop() readonly usageType!: string;
+    @Prop() readonly brand!: string;
```

Comment:
non needed

Reply:
I need it to the updateSupplies - I will change it later